### ISSUE TITLE
Inspector Doesn't Indicate Percentage of Transmission 4.0.0 Peer

### ISSUE THREAD

**Issue by Tockman** [transmission/transmission#5078]
### What is the issue?

![Снимок экрана 2023-02-27 в 17 59 57](https://user-images.githubusercontent.com/74548419/221599310-7425a530-81f9-4e79-a3fd-51e9108234a9.png)
Either the inspector doesn't indicate or client doesn't send data, just like Xunlei.

### Which application of Transmission?

macOS app

### Which version of Transmission?

4.0.1 (e1c6e1be43)

**Comment by Coeur** [comment]
@ckerr I don't think it qualifies it as a mac issue: if there is any bug for that, it would likely be in core code only, since:
a. We can't assert that the Transmission peer is using the mac app or a qt app or a gtk app or a web remote client.
b. The GUI is only displaying whatever value is given from libtransmission.

**Comment by Tockman** [comment]
Looks like this problem's solved already in 4.0.1. 
![Снимок экрана 2023-03-19 в 19 09 50](https://user-images.githubusercontent.com/74548419/226189196-c087ebbd-45ec-449f-b39f-0b2bc2c56b36.png)
